### PR TITLE
CameraRigComponent - Disable Component Checkbox when the two others are toggled

### PR DESCRIPTION
**Pull request by AMZN-Igarri**
This is a followup PR to #5410 
This disables the checkbox when the other two are activated so you cannot pass a zero vector.

https://user-images.githubusercontent.com/82394219/141302570-830c122d-e3e2-45d5-8aa6-47fc26a9df8a.mp4

### CODE REVIEW

**Review by hultonha — APPROVED** [review]
This looks great! 😄 Thanks for including the video too, really nice! 🙂

**Review by tjmichaels — APPROVED** [review]

File: Gems/StartingPointCamera/Code/Source/CameraLookAtBehaviors/SlideAlongAxisBasedOnAngle.h [hunk 1]
```diff
@@ -23,8 +23,7 @@ namespace Camera
     /// will occur when looking up.  This could also be used for peeking around
     /// corners.  This is primarily useful for third person cameras.
     //////////////////////////////////////////////////////////////////////////
-    class SlideAlongAxisBasedOnAngle
-        : public ICameraLookAtBehavior
+    class SlideAlongAxisBasedOnAngle : public ICameraLookAtBehavior
```
Comment (cgalvan) [inline Gems/StartingPointCamera/Code/Source/CameraLookAtBehaviors/SlideAlongAxisBasedOnAngle.h:26]: I believe our coding standard is for the public to be on a newline like it was before, unless that has changed?

```
class SlideAlongAxisBasedOnAngle
        : public ICameraLookAtBehavior
```

Comment (hultonha) [reply Gems/StartingPointCamera/Code/Source/CameraLookAtBehaviors/SlideAlongAxisBasedOnAngle.h]: `clang-format` will stack them when there's more than one but puts them on the same line when there's only one, I think this is something worth living with to get auto-formatting so I'd say it's okay to leave as-is

**Comment by amzn-leenguy** [comment]
Good work and thanks for the video

**Review by bhanuja-s — COMMENTED** [review]
Looks good, thanks for adding me to review!

**Review by cgalvan — APPROVED** [review]

**Pull request merged**